src/sentry/static/sentry/app/components/pageHeader.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+class PageHeader extends React.Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+  };
+
+  render() {
+    const {children, className} = this.props;
+    return <Wrapper className={className}>{children}</Wrapper>;
```

Comment:
The downside to something like that, is storybook will look like this:

![image](https://user-images.githubusercontent.com/435981/50119962-14e4a200-0209-11e9-84a4-133b8521a47f.png)

Vs with the way it is right now:

![image](https://user-images.githubusercontent.com/435981/50120090-4c534e80-0209-11e9-9006-fcfb2bbf7a42.png)

We will also get somewhat confusing compiled classnames in dev and snapshots:

![image](https://user-images.githubusercontent.com/435981/50120548-b9b3af00-020a-11e9-8114-5ec40a79044f.png)

This woulds be particularly confusing if a developer wanted to write `styled(PageHeader)` in a parent component.

We could solve the snapshot issue by renaming the base component to something like `UnstyledPageHeader`, and then maybe we could figure out a way to make storybook play better with raw SCs, but I guess I'm curious how much benefit we get from building selectors with imported styled components anyways? I worry it will retain a lot of the tangled dependency issues that css nesting contained.

Reply:
Ugh the storybook issue is a deal breaker. 

I don't think the compiled class names are a big issue (and no one actually reads snapshots anyway).

> but I guess I'm curious how much benefit we get from building selectors with imported styled components anyways?

This is a good point, we should make SC selectors an exception and not the rule since they shouldn't be used frequently.

Reply:
To get proptypes back on our styled component in storybook, can we just assign the proptypes property?

```js
StyledPageHeader.propTypes = PageHeader.propTypes;
```